docs/topics/governance-legal/index.md

```diff
@@ -35,14 +35,14 @@ Genetic data generated in a healthcare context is subject to more stringent info
 
 ### Use cases
 
-Understand more about the current [use cases and architecture for the Federated EGA](https://zenodo.org/record/4893063) as reported by the inaugural Federated EGA nodes.
+Understand more about the current [use cases and architecture for the Federated EGA](https://zenodo.org/record/4893063). Here you can read about the requirements and national use cases of early engaged Federated EGA Nodes which aim to be addressed by joining the Federated EGA. The report also describes how specific use cases can be supported by Federated EGA, for example by a governance structure made up of permanent committees and ad hoc working groups, the use of global and community standards, and guidelines for how data/metadata can be shared within the network. 
```

Comment:
```suggestion
Understand more about the current [use cases and architecture for the Federated EGA](https://zenodo.org/record/4893063). Here you can read about the requirements and national use cases of early engaged Federated EGA Nodes which aim to be addressed by joining the Federated EGA. The report also describes how specific use cases can be supported by Federated EGA. For example, by a governance structure made up of permanent committees and ad hoc working groups, the use of global and community standards, and guidelines for how data/metadata can be shared within the network. 
```

Reply:
I prefer to keep this as is to avoid incomplete sentences.

Reply:
The phrase just seemed a bit too long, but it's understandable anyway